GOVERNANCE.md

```diff
@@ -0,0 +1,96 @@
+# Project governance
+
+The tldr-pages project strives to have an **open**, **welcoming**,
+and [**non-hierarchical**](https://en.wikipedia.org/wiki/Flat_organization)
+governance structure.
+
+To that end, this document describes the principles
+that guide the self-management of the project.
+By having them written down explicitly, and open to scrutiny,
+the entire community can read, improve and adapt them as needed,
+with no central authority.
+
+
+## I. Participation and community interactions
+
+- **All contributions are welcome**,
+  [no matter how small](https://github.com/kentcdodds/all-contributors).
+  The tldr-pages project is a [do-ocracy](https://communitywiki.org/wiki/DoOcracy),
+  so don't hesitate to get involved — we're happy to welcome you into the community!
+  Please take a look at [CONTRIBUTING.md](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md)
+  to get started.
+
+- All members of the community are expected to **be cordial in all communications**.
+  Avoid making assumptions about the others' intentions, and make your own intentions clear.
+  When in doubt, provide additional context, or ask for clarification.
+  Remember, it's very hard to convey meaning on a purely written medium,
+  especially between people from different cultures, technical backgrounds,
+  English proficiency levels, etc.
+
+- **All communications are public**.
+  There are no permanent private channels where maintainers discuss "internal" matters.
+  Occasional private chat or email messages may be exchanged,
+  e.g. when setting up services that require passwords,
+  but otherwise all communications that impact the project
+  will either happen in issue and PR discussions,
+  or in the [Gitter chat room](https://gitter.im/tldr-pages/tldr)
+  (which is open to all, and publicly logged).
+
+- **All decisions are made by community consensus.**
```

Comment:
Yeah, I was a little liberal with the terminology here, but the fuzziness is intentional (hence the "ideally" below). The rationale for this choice of words is (1) I assumed most people would be more likely to get the right idea from "community consensus" than "community consent", and (2) I wanted to give the community some wiggle room, rather than hint to a strict specification of the consent method as a rule, since it can be abused as some sort of veto, which can cause friction and bottlenecks in discussions.

Does this make sense to you?